src/main/java/uk/gov/ons/census/action/schedule/FulfilmentProcessor.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.ons.census.action.schedule;
+
+import java.util.List;
+import java.util.UUID;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.census.action.model.entity.FulfilmentMapper;
+
+@Component
+public class FulfilmentProcessor {
+
+  private JdbcTemplate jdbcTemplate;
+
+  public FulfilmentProcessor(JdbcTemplate jdbcTemplate) {
+    this.jdbcTemplate = jdbcTemplate;
+  }
+
+  public void addFulfilmentBatchIdAndQuantity() {
+    String query =
+        "select fulfilment_code, count(*) from actionv2.fulfilment_to_send group by fulfilment_code";
```

Comment:
strictly speaking, this is a string constant and could be defined as a `private static final` member variable